content/3_Silver/Func_Graphs.mdx

```diff
@@ -716,6 +716,71 @@ print(" ".join(str(i + 1) for i in ans))
 </PySection>
 </LanguageSection>
 
+### Solution 3
+
+It's also possible to use `floyd(x)` to generate answers for all vertices in the connected component
+containing `x` without using adjacency lists.
```

Comment:
what do you use instead of adjacency lists then? explain here

Reply:
wait hold on i read your other comment- what's the complexity of this solution?

Reply:
It seems, I have caused a bit of chaos and confusion. Please ignore the previous pr, which I had earlier mentioned.

My approach:
So, I am using floyd, to find the cycles and then update the coordinates of non cycles and cycle vertices. While, finding a cycle from a, if we encounter ans[b] to not be -1, this means, this coordinate has been already calculated. In this case, the fillUp() method is called. There are two scenarios, either this coordinate (b) is in a cycle, or a non cycle. Whatever might be the case, from that node a to this b, the, answer will for all of the nodes will be ans[b].  

I hope, I was clear.

Reply:
> however, its not always trusted...

wdym by this

Reply:
i meant, the runtime metric no. shown by the online judges in general is not trustable (specially leetcode). have been advised by people around, and blogs in general. personally, i have found a lot of time discrepancy on leetcode, but not so much on codeforces. for this problem, i ran it twice today, and it gave a 30ms difference. not that much, but still. so, i don't focus too much on that runtime metric number.